client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/RssShuffleManager.java

```diff
@@ -53,7 +54,8 @@ public class RssShuffleManager implements ShuffleManager {
   public RssShuffleManager(SparkConf conf) {
     this.conf = conf;
     this.celebornConf = SparkUtils.fromSparkConf(conf);
-    this.cores = conf.getInt(SparkLauncher.EXECUTOR_CORES, 1);
+    this.executorCores = conf.getInt(SparkLauncher.EXECUTOR_CORES, 1);
+    this.driverCores = conf.getInt("spark.driver.cores", 0);
```

Comment:
I suggest renaming `conf: SparkConf` to `sparkConf: SparkConf`, and renaming `celebornConf: CelebornConf` to `conf: CelebornConf`, to be consistent w/ other places.